src/reusecore/PageHeader/index.js

```diff
@@ -45,10 +45,10 @@ const PageHeader = ({ category, title, img, feedlink, subtitle, author, thumbnai
   return (
     <PageHeaderWrapper>
       <div className="page-header">
-        { thumbnail && <div className="feature-image">
-          <Image {...thumbnail} imgStyle={{ objectFit: "contain" }} alt={title}/>
+        {thumbnail && <div className="feature-image">
+          <Image {...thumbnail} imgStyle={{ objectFit: "contain" }} alt={title} />
```

Comment:
Same here. If you're pressing for a review, then make it as easy as possible on the maintainers, who can otherwise spend time reviewing meaningful changes (changes that have an impact on the site's behavior or appearance).